docs/spec/v1beta2/receivers.md

```diff
@@ -0,0 +1,914 @@
+called.
+
+A resource entry contains the following fields:
+
+- `apiVersion`: The Flux Custom Resource API group and version, such as
+  `source.toolkit.fluxcd.io/v1beta2`.
+- `kind`: The Flux Custom Resource kind, supported values are `Bucket`,
+  `GitRepository`, `Kustomization`, `HelmRelease`, `HelmChart`,
+  `HelmRepository`, `ImageRepository`, `ImagePolicy`, `ImageUpdateAutomation`
+  and `OCIRepository`.
+- `name`: The Flux Custom Resource `.metadata.name`. This field may contain
+  a wildcard `*` to match all resources of the `kind`.
+- `namespace` (Optional): The Flux Custom Resource `.metadata.namespace`.
+  When not specified, the Receiver's `.metadata.namespace` is used instead.
+
+
+**Note:** Cross-namespace references [can be disabled for security
+reasons](#disabling-cross-namespace-selectors).
+
+### Secret reference
+
+`.spec.secretRef.name` is a required field to specify a name reference to a
+Secret in the same namespace as the Receiver. The Secret must contain a `token`
+key, whose value is a string containing a (random) secret token.
+
+This token is used to salt the generated [webhook path](#webhook-path), and
+depending on the Receiver [type](#supported-receiver-types), to verify the
+authenticity of a request.
+
+#### Secret example
+
+```yaml
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: webhook-token
+  namespace: default
+type: Opaque
+stringData:
+  token: <random token>
+````
+
+### Interval
+
+`.spec.interval` is a required field with a default of ten minutes that specifies
+the time interval at which the controller reconciles the provider with its Secret
+reference.
+
+### Suspend
+
+`.spec.suspend` is an optional field to suspend the Receiver.
+When set to `true`, the controller will stop processing events for this Receiver.
+When the field is set to `false` or removed, it will resume.
+
+## Working with Receivers
+
+### Disabling cross-namespace selectors
+
+On multi-tenant clusters, platform admins can disable cross-namespace
+references with the `--no-cross-namespace-refs=true` flag. When this flag is
+set, Receivers can only refer to [Resources](#resources) in the same namespace
+as the [Alert](alerts.md) object, preventing tenants from triggering
+reconciliations to another tenant's resources.
+
+### Public Ingress considerations
+
+Considerations should be made when exposing the controller's `webhook-receiver`
+Kubernetes Service to the public internet. Each request to a Receiver [webhook
+path](#webhook-path) will result in request to the Kubernetes API, as the
+controller needs to fetch information about the resource. This endpoint may be
+protected with a token, but this does not defend against a situation where a
+legitimate webhook caller starts sending large amounts of requests, or the
+token is somehow leaked. This may result in unwanted consequences for the
+controller, as it may get rate limited by the Kubernetes API, degrading its
+functionality.
```

Comment:
Maybe?
```suggestion
token is somehow leaked. This may result in the controller, as it may get rate limited
by the Kubernetes API, degrading its functionality.
```